### PR TITLE
Remove topics pages with zero items 

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

This PR will remove topics pages that have been listed with zero items 

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-topics-removal/)

### Solution

Deleted folders that were listed in the spreadsheet with zero items.

### How To Test

Add the following paths to the preview link and ensure they are giving a 404 error 

1. `topics/cops-office`
2. `topics/defense-intelligence-agency`
3. `topics/dgu`
4. `topics/dia`
5. `topics/digitalgov-university`
6. `topics/first-responder-network-authority`
7. `topics/first-responders-network`
8. `topics/national-museum-of-african-american-history-and-culture`
9. `topics/nmaahc`
10. `topics/office-of-community-oriented-policing-services`
11. `topics/office-of-inspector-general`
12. `topics/oig`
13. `topics/us`
14. `topics/usfs`
15. `topics/whats-new`
16. `topics/work-in-progress`

---

### Dev Checklist

- [ ] PR has correct labels
- [ ] A11y testing (voice over testing, meets WCAG, run axe tools)
- [ ] Code is formatted properly

### CODE REVIEW

File: content/resources/an-introduction-to-trust.md
Comment (clmedders) [inline content/resources/an-introduction-to-trust.md:1]: this file was fixed from linting

**Review by mejiaj — APPROVED** [review]
Confirming directories are removed. Found no references to these tags in frontmatter.

**Review by nick-mon1 — APPROVED** [review]
Removed topics return 404 and did not see these removed tags in the markdown front matter. Thanks!

**Review by bonnieAcameron — APPROVED** [review]
looks good to me!

**Comment by ToniBonittoGSA** [comment]
I saved each to Wayback; https://web.archive.org/

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**